doc/train/finetuning.md

```diff
@@ -45,3 +47,121 @@ To obtain a more simplified script, for example, you can change the {ref}`model
         "fitting_net" : {}
     }
 ```
+
+## PyTorch Implementation {{ pytorch_icon }}
+
+In PyTorch version, leveraging the flexibility offered by the framework and the multi-task training capabilities provided by DPA2 [paper](https://arxiv.org/abs/2312.15492),
```

Comment:
by saying " multi-task training capabilities " and referring to DPA-2 paper,   the reader may think you assume that the model is pretrained by multi-task in this section